dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -341,6 +341,6 @@ public void testIsUnusedClassSymbolReference_classSymbolReferenceNotFound()
             .setSubclass(false)
             .setTargetClassName("dummy.NoSuchClass")
             .build();
-    Truth.assertThat(classDumper.isUnusedClassSymbolReference(referenceToUnusedClass)).isTrue();
+    Truth.assertThat(classDumper.isUnusedClassSymbolReference(referenceToUnusedClass)).isFalse();
```

Comment:
I thought you made it into an error now?

Reply:
returning false for "isUnused?" meaning this is used. So the reference is reported as an error.